docs/requirements.txt

```
@@ -2,5 +2,6 @@ docutils
 m2r2
 nbsphinx
 sphinx-copybutton
-sphinx>=3.1.0
+sphinx==4.5.0
```

What is the motivation behind the strict version constraint?

The dependencies in `docs/requirements.txt` should be as _loose_ as possible as they are not captured in setup.py.

---

Ah, gotcha! In that case, let's place an upper bound on the Sphinx version to v5.0.0 to be as flexible as possible:
```suggestion
sphinx<5.0.0
```